### PR TITLE
 Topbar with filters

### LINKED ISSUE #2223: Topbar with filters
1. Trigger current filter dialog for adding filters
2. Show applied filters on the topbar as string (like currently, new statusbar in another issue - https://github.com/taskany-inc/issues/issues/2350)
3. Support saving presets
4. Search is contextual

### PR DESCRIPTION
**Pull request by asabotovich**
## PR includes

- [x] Feature

## Related issues

Resolve #2223 

## QA Instructions, Screenshots, Recordings

Screenshot 2024-03-05 at 12 49 40

### CODE REVIEW

File: src/components/FiltersBar/FiltersBar.module.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.FiltersBarCounter {
+    display: flex;
+    align-items: center;
+    color: var(--text-secondary);
+}
+
+.FiltersBarCounterActive:after {
+    content: '\a0';
```
Comment (awinogradov) [inline src/components/FiltersBar/FiltersBar.module.css:64]: What is it?

File: src/components/FiltersBar/FiltersBar.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    </div>
+);
+
+export const FiltersBarControlGroup: FC<HTMLAttributes<HTMLDivElement>> = ({ children, className, ...props }) => (
+    <div className={cn(s.FiltersBarControlGroup, className)} {...props}>
+        {children}
+    </div>
+);
+
+export const FiltersBarViewDropdow: FC = () => (
```
Comment (awinogradov) [inline src/components/FiltersBar/FiltersBar.tsx:91]: Dropdow`N`

File: src/utils/getPageTitle.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const getPageTitleTitle = ({
```
Comment (awinogradov) [inline src/utils/getPageTitle.ts:1]: TitleTitle? WhyWhy?

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/components/FiltersBar/FiltersBar.tsx (hunk 2 above)
Comment (asabotovich) [reply src/components/FiltersBar/FiltersBar.tsx]: fixed

File: src/components/FilteredPage/FilteredPage.tsx [hunk 4]
```diff
@@ -36,61 +27,46 @@ interface FilteredPageProps {
...
-                    id: currentPreset.id,
-                    direction: !currentPreset._isStarred,
-                });
-                await onFilterStar();
-            }
-        } else {
-            dispatchModalEvent(ModalEvent.FilterCreateModal)();
-        }
-    }, [currentPreset, toggleFilterStar, onFilterStar]);
+    // TODO: https://github.com/taskany-inc/issues/issues/2413
```
[the remark this answers is not in the document]
Comment (asabotovich) [reply src/components/FilteredPage/FilteredPage.tsx]: fixed

**Review by awinogradov — APPROVED** [review]

**Review by DenisVorop — APPROVED** [review]

**Pull request merged**